src/test/shared/systemUtilities.test.ts

```diff
@@ -122,4 +124,65 @@ describe('SystemUtilities', function () {
         const regex = /bin[\\\/](code|code-insiders)$/
         assert.ok(regex.test(vscPath), `expected regex ${regex} to match: "${vscPath}"`)
     })
+
+    describe('permissions', function () {
+        beforeEach(function () {
+            if (process.platform === 'win32') {
+                this.currentTest?.skip()
+            }
```

Comment:
Need to investigate how permissions are modeled on Windows. None of the tests create errors in our CI job, maybe because we're running as admin?

Reply:
> Or libuv simply doesn't implement mode on Windows given the differences in permissions modeling.

I think that's it. Windows ntfs has ACLs.

So we might want to test things by trying to actually read/write a dummy file instead of trying to inspect the attributes.